snuba/settings_base.py

```diff
@@ -22,6 +22,7 @@
 
 # Redis Options
 USE_REDIS_CLUSTER = False
+REDIS_CLUSTER_STARTUP_NODES = None
```

Comment:
Since `USE_REDIS_CLUSTER` defaults to `False`, I think it might make more sense to leave this out of the base configuration and use `getattr` instead in `redis.py` (unless we really think that many people will use it?)

Reply:
That's fine too, it was my initial impression but seemed nice to have it here at least as a form of some documentation that the value exists.

Reply:
Makes sense, I think you could make a reasonable argument either way.